src/erlcloud_ec2.erl

```diff
@@ -2302,6 +2302,9 @@ extract_volume(Node) ->
      {status, get_text("status", Node, none)},
      {create_time, erlcloud_xml:get_time("createTime", Node)},
      {volumeType, get_text("volumeType", Node)},
+     {iops, get_integer("iops", Node)},
+     {encrypted, get_bool("encrypted", Node)},
+     {kms_key_id, get_text("kmsKeyId", Node, none)},
```

Comment:
the only reason is these new properties are optional and their default value (`none`) is the same as for the other properties in `volume` object (see `status`, for example). Please note that `iops` is optional as well, and it is defaulted to `0` by `erlcloud_xml: get_integer/2,3`, please see https://github.com/erlcloud/erlcloud/blob/master/src/erlcloud_xml.erl#L100-L105